src/main/java/uk/gov/hmcts/divorce/common/notification/FinancialOrderRequestedNotification.java

```diff
@@ -0,0 +1,123 @@
+            populateTemplateVars(caseData, caseId, caseData.getApplicant1(), caseData.getApplicant2());
+
+        notificationService.sendEmail(
+            email,
+            FINANCIAL_ORDER_REQUESTED_NOTIFICATION,
+            templateVars,
+            languagePreference,
+            caseId
+        );
+    }
+
+    @Override
+    public void sendToApplicant2(final CaseData caseData, final Long caseId) {
+
+        final String email = caseData.getApplicant2EmailAddress();
+        final LanguagePreference languagePreference = caseData.getApplicant2().getLanguagePreference();
+
+        final Map<String, String> templateVars =
+            populateTemplateVars(caseData, caseId, caseData.getApplicant2(), caseData.getApplicant1());
+
+        if (caseData.getApplicationType().isSole()) {
+            if (isNotBlank(email)) {
+                log.info("Sending financial order requested notification to respondent for case : {}", caseId);
+
+                notificationService.sendEmail(
+                    email,
+                    RESPONDENT_FINANCIAL_ORDER_REQUESTED_NOTIFICATION,
+                    templateVars,
+                    languagePreference,
+                    caseId
+                );
+            } else {
+                log.info("Respondent email is blank. Financial Order Notification will not be sent.");
+            }
+        } else {
+            log.info("Sending financial order requested notification to applicant 2 for case : {}", caseId);
+
+            notificationService.sendEmail(
+                email,
+                FINANCIAL_ORDER_REQUESTED_NOTIFICATION,
+                templateVars,
+                languagePreference,
+                caseId
+            );
+        }
+    }
+
+    public Map<String, String> populateTemplateVars(final CaseData caseData,
+                                                    final Long id,
+                                                    final Applicant applicant,
+                                                    final Applicant partner) {
+        final Map<String, String> templateVars =
+            commonContent.mainTemplateVars(caseData, id, applicant, partner);
+
+        if (YesOrNo.YES.equals(applicant.getFinancialOrder())) {
+            templateVars.put(FINANCIAL_ORDER_REQUESTED,YES);
+            templateVars.put(FINANCIAL_ORDER_NOT_REQUESTED,NO);
+        } else {
+            templateVars.put(FINANCIAL_ORDER_REQUESTED,NO);
+            templateVars.put(FINANCIAL_ORDER_NOT_REQUESTED,YES);
+        }
+
+        String financialOrderFees = formatAmount(paymentService.getServiceCost(SERVICE_OTHER, EVENT_MISC, KEYWORD_FINANCIAL_ORDER_NOTICE));
+        templateVars.put(FEES_FINANCIAL_ORDER, financialOrderFees);
+        String consentOrderFees = formatAmount(paymentService.getServiceCost(SERVICE_OTHER, EVENT_GENERAL, KEYWORD_CONSENT_ORDER));
+        templateVars.put(FEES_CONSENT_ORDER, consentOrderFees);
+        return templateVars;
```

Comment:
I'm not sure of the wording in the template, but is it worth putting in some logic around these api calls so they are only triggered when necessary?  (ie, Do we need to get the financialOrderFees if there is no Financial order?)

Reply:
Ok - so I'm confused by line 109.  If the applicant hasn't requested a final order, why do we need to get the fee value?  Do we always need to get both the financial order fee and the consent order fee?

Reply:
@BWeeks-JusticeGovUK The additional condition check that you have mentioned is to allow the notification content to be rendered based on which applicant has applied for financial order